test/helpers.py

```diff
@@ -89,3 +112,166 @@ def _wait_until_query_success(
             f"Query ID: {sfqid}"
             f"Final query status: {status}"
         )
+
+
+def create_pyarrow_iterator(input_data, use_table_iterator):
```

Comment:
Is it the caller's responsibility to make sure arrow related iterators are importable?

Reply:
yes, it's the test case's responsibility